orderbook/src/database/events.rs

```diff
@@ -152,6 +161,11 @@ async fn delete_events(
         .execute(sqlx::query(QUERY_SETTLEMENTS).bind(delete_from_block_number as i64))
         .await?;
 
+    const QUERY_PRESIGNATURES: &str = "DELETE FROM presignature_events WHERE block_number >= $1;";
```

Comment:
Could we call this constant something with the word "DELETE" in it? Maybe `DELETE_PRESIGNATURE_QUERY` or `QUERY_DELETE_PRESIGNATURES`

Reply:
This is an implementation of the function `delete_events`, so I think it's ok if no `DELETE` is used in the variable. Note that the surrounding code also follows the syntax `QUERY_*`, so having all variables read `QUERY_DELETE_*` would be a bit redundant.